src/Zope2/utilities/tests/test_zconsole.py

```diff
@@ -92,6 +92,6 @@ def test_runscript(self):
         # so we have the raw string in there.
         expected = (
             "42\n"
-            r"['run', {!r}, {!r}, {!r}, 'baz']"
-            "\nPropertyManager\n".format(self.zopeconf, script, bar))
+            rf"['run', {self.zopeconf!r}, {script!r}, {bar!r}, 'baz']"
```

Comment:
Hm, I wonder with the `r"..."` is needed here.

Reply:
https://github.com/zopefoundation/Zope/blob/058107d60fcd7b0bb461b8dfb4ead7257236a35f/src/Zope2/utilities/tests/test_zconsole.py#L91-L92 has the explanation